ledger/appcow_test.go

```diff
@@ -418,6 +419,18 @@ func TestCowBuildDelta(t *testing.T) {
 		ed,
 	)
 
+	// check v27 behavior
+	cow.proto = config.Consensus[protocol.ConsensusCurrentVersion]
```

Comment:
if you want a specific behavior, you should call it by name; There is no reason to avoid using the protocol.V27